doc/tutorials/cascading.md

```diff
@@ -91,41 +91,16 @@ The **proc** namespace is not accessible by the command line tool **kdb**, as it
 
 The **spec** namespace is used to store metadata about keys and therefore Elektra handles the **spec** namespace differently to other namespaces. The following part of the tutorial is dedicated to the impact of the **spec** namespace on cascading lookups.
 
-## Write Operations and the cascading Namespace
+## Cascading writes are not possible
 
 If a value is to be written to a cascading key, i.e., a key starting with '/', only cascading keys that resolve to an existing key will be used.
```

Comment:
This sentence should be changed too. Right now it implies that this would work:

```c
kdb set user:/foo 123
kdb set /foo 456
```

I think that was the case at some point, but judging by the code is no longer allowed.